### DISCUSSION TITLE
Business

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#330] [General]
I am starting a new business in a couple of weeks. Any tips or recommendations would you like to share? What are the most effective ads in your opinion?

**Comment by XelaAl** [comment]
Here are some tips:
1. Start on a small scale before going all out. 
2. Don't fixate on mistakes or get demoralized by them. 
3. Learn from others. 
4. Think of what you do AS a business. 
5. Understand the difference between working for yourself and building an ongoing business.

**Comment by hernessteve** [comment]
Do you know [what is product matching](https://forbytes.com/blog/product-matching-in-ecommerce/)? I highly recommend you find more infos about it cause marketing strategy is one of the most important components in every single business. Every single business owner (and the future one too) should pay attention on that
